pincer/client.py

```diff
@@ -339,17 +336,25 @@ async def on_ready(self):
                 "it gets treated as a command and can have a response."
             )
 
-        _events[name].append(coroutine)
-        return coroutine
+        event = InteractableStructure(
+            call=coroutine
+        )
+
+        _events[name].append(event)
+        return event
 
     @staticmethod
-    def get_event_coro(name: str) -> List[Optional[Coro]]:
+    def get_event_coro(name: str) -> List[Optional[InteractableStructure[None]]]:
```

Comment:
we need a shorten name for the type
cause right now
```py
>>> len("List[Optional[InteractableStructure[None]]]")
43
```

Reply:
It can be shortened to 39 like this.
`List[None, InteractableStructure[None]]`

Reply:
Could also rename the class to `InteractableStruct`

Reply:
Not this pr